wordpress-mobile/gutenberg-mobile
# Improve unsupported blocks

Show localized title and icon of unsupported core blocks. It also drops our custom `gmobile/unsupported` block and moves the functionality to a native variant of `core/missing`.

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/14577

![Simulator Screen Shot - iPhone XS - 2019-03-21 at 11 06 30](https://user-images.githubusercontent.com/8739/54745485-e9879f80-4bc9-11e9-9d5d-4d5bb54a6841.png)

To test:

- Create a post with some blocks that are unsupported in the app.
- Open it in gutenberg-mobile and verify that the blocks show the right icon and localized block title. For non-core blocks, a generic icon is shown.

Part of #479
